packages/@aws-cdk/aws-servicecatalog/lib/common.ts

```diff
@@ -17,3 +18,28 @@ export enum AcceptLanguage {
    */
   ZH = 'zh'
 }
+
+/**
+ * Tag Option Value
+ */
+interface TagOptionValue {
+  /**
+   * The tag value of the key-value pair.
+   */
+  readonly value: string;
+
+  /**
+   * Active status of option. By default, a tagOption is
+   * active upon creation, and can only be disabled on an update.
+   * @default - true
+   */
+  readonly active?: boolean;
+}
+
+/**
+ * Defines a Tag Option, which are similar to tags
+ * but have multiple values per key.
+ */
+export interface TagOption {
+  readonly [key: string]: TagOptionValue[];
+}
```

Comment:
Commented above on an alternative design for this concept.

Reply:
punting to new PR